README.md

```diff
@@ -1,141 +1,132 @@
 # create-openapi-repo
 
-[![NPM version][npm-image]][npm-url] [![Dependency Status][daviddm-image]][daviddm-url]
-
-> Generate an organized multi-file OpenAPI repository.
+A tool for generating multi-file OpenAPI definitions.
 
-<center>
+[![NPM version][npm-image]][npm-url] [![Dependency Status][daviddm-image]][daviddm-url]
 
+<p align="center">
 <img src="./logo.png" width="500px"/>
+</p>
 
-</center>
-
-## Who?
-You! Hello.  Do you need to write or contribute to an OpenAPI definition?  If so, read on...
-
-## What?
-We recommend a docs-like-code approach to OpenAPI definitions:
-- Write it using your favorite text-editor or IDE (we love VSCode).
-- Organize it into multiple files and folders to make it easy to navigate.
-- Store it using source control (such as GitHub).
-- Continuously validate it using our free [openapi-cli tool](https://github.com/redocly/openapi-cli) or our free continuous validation service (coming soon).
-- Bundle it (for a smaller footprint to use in other tools or for tools that do not support a multi-file format).
-
-## Why?
-There are a few advantages in hosting your API definition on GitHub:
- - Community engagement (PR's and issues -- if you have a public repo)
- - Advertisment in the GitHub community
- - Hosting on GitHub pages (perfect uptime, CDN, Jekyll, custom domains with CNAME)
- - Revision history, branching, CI
- - Review and approval workflows using Pull Requests
- - Fast on-boarding time (developers and tech writers know how to use GitHub :smile:)
- - Fully compatible with Redocly API Reference too
-
- There are also some advantages to a multi-file YAML format OpenAPI definition:
- - Reuse schema objects to keep things DRY (don't repeat yourself) 
- - Smaller diffs compared to JSON (especially for markdown descriptions)
- - Easier to navigate
- - Easier to edit with confidence
+Need to write or contribute to an OpenAPI definition? `create-openapi-repo` can help with that!
 
 ## Features
-This generator helps to create a GitHub repo with the following features:
- - Split a big (or small) OpenAPI definition into smaller files organized into folders
- - Bundle it into a single file for deployment
- - Continuous integration/deployment on Travis or Redocly Workflows
- - Code samples as separate files
- - Automate deployment of your OpenAPI definition and docs
- - OpenAPI definition is validated after each commit
+
+ - OpenAPI 3.0 support
+ - Split an existing OpenAPI definition into multiple files
+ - Bundle a multi-file definition into a single file
+ - Validate your OpenAPI definition using our free [openapi-cli tool](https://github.com/redocly/openapi-cli)
+ - Automate deployment of your API reference docs using CI/CD workflows
+ - Maintain code samples as separate files
  - Live editing in your editor of choice :heart_eyes:
 
-## Structure
+## Examples
```

Comment:
not sure about the last two examples - one is archived and the other hasn't been touched in seven years